.claude/skills/codebase-audit/SKILL.md

```
@@ -3625,13 +3804,23 @@ For each finding below, do:
 
 Write results to: _audit/latest/findings/validate-batch-{N}.md
 
-Format per finding:
+Format per finding (mechanically parseable; synthesizer reads this verbatim):
 ### [original-file]:[line] -- [CONFIRMED|FALSE_POSITIVE|INTENTIONAL]
 **Original**: [description from audit agent]
-**Actual code**: [quoted code]
-**Verdict**: [explanation]
+**Actual code**:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)